cmd/network/status.go

```diff
@@ -132,7 +135,22 @@ var (
 					fmt.Println("==== ParaTimes ====")
 				}
 
-				for _, runtime := range nodeStatus.Runtimes {
+				keys := make([][]interface{}, 0, len(nodeStatus.Runtimes))
```

Comment:
I would just store the map keys in the `keys` array. The sorting logic (ID vs name) should all come in the `sort.Slice()` function.

```suggestion
				keys := make([]string, 0, len(nodeStatus.Runtimes))
```